src/coreclr/jit/compiler.h

```diff
@@ -5958,17 +5960,10 @@ class Compiler
 
     // Do hoisting of all loops nested within loop "lnum" (an index into the optLoopTable), followed
     // by the loop "lnum" itself.
-    //
-    // "m_pHoistedInCurLoop" helps a lot in eliminating duplicate expressions getting hoisted
```

Comment:
Any reason this comment was deleted?

Reply:
Looks like an inadvertent edit -- at one point I had modified the signature of optHoistLoopNest.